configs/AM57X/AM57X_linux_config.txt

```diff
@@ -24,6 +24,7 @@ Replacement Variables
 '__SDK_DOWNLOAD_URL__'           : '`AM57X-Linux-SDK-Download-page <http://software-dl.ti.com/processor-sdk-linux/esd/AM57X/latest/index_FDS.html>`__'
 '__LINUX_UBUNTU_VERSION_LONG__'  : '18.04 (64-bit)'
 '__LINUX_UBUNTU_VERSION_SHORT__' : '18.04'
+'__OPTEE_PLATFORM_FLAVOR__'      : 'am57x'
```

Comment:
Isn't it `am57xx` ?

https://github.com/OP-TEE/optee_os/blob/b8125477b03b73ceeccc65c33e4b8cd93d3fff1a/core/arch/arm/plat-ti/conf.mk#L13

Reply:
Yeah, it should be am57xx

@jeevantelukula please plan to update it